packages/indy-vdr/src/IndyVdrModuleConfig.ts

```diff
@@ -0,0 +1,21 @@
+export interface IndyVdrNetworkConfig {
+  indyNamespace: string
+  genesisTransaction: string
+  isProduction: boolean
+}
+
+export interface IndyVdrModuleConfigOptions {
+  networks: IndyVdrNetworkConfig[]
```

Comment:
Some documentation you can reuse from the indy sdk module:

```
  /**
   * Array of indy networks to connect to.
   *
   * @default []
   *
   * @example
   * ```
   * {
   *   isProduction: false,
   *   genesisTransactions: 'xxx',
   *   indyNamespace: 'localhost:test',
   *   transactionAuthorAgreement: {
   *     version: '1',
   *     acceptanceMechanism: 'accept'
   *   }
   * }
   * ```
   */
  networks?: IndySdkPoolConfig[]
```